Makefile

```diff
@@ -105,7 +105,10 @@ docker-build: ## Build and start the container cluster
 docker-up: ## Start the container cluster
 	@docker compose up -d
 
-docker-down: ## Start the container cluster
+docker-stop: ## (Only) stop the container cluster, without removing the containers
```

Comment:
For consistency purpose I prefer to keep the same structure as the other comments with "Stop the container cluster".
That way we would have:
```
## Build and start the container cluster
## Start the container cluster
## Stop the container cluster
## Stop and remove the container cluster
```

It may be a less precautionary way to label that but as @tchoutri : let's keep it as close as the docker-compose experience.